apps/core-app/src/legos/tx.ts

```diff
@@ -270,3 +271,67 @@ export const TX: Record<string, TXLego> = {
     ],
   },
 };
+
+export const TABULA_TX: Record<string, TXLego> = {
+  CREATE_PUBLICATION: buildMultiCallTX({
+    id: 'CREATE_PUBLICATION',
+    JSONDetails: {
+      type: 'JSONDetails',
+      jsonSchema: {
+        title: '.formValues.title',
+        description: '.formValues.description',
+        link: '.formValues.link',
+        proposalType: { type: 'static', value: 'Create Publication Proposal' },
+      },
+    },
+    actions: [
+      {
+        contract: CONTRACT.POSTER,
+        method: 'post',
+        args: [
+          {
+            type: 'JSONDetails',
+            jsonSchema: {
+              action: { type: 'static', value: TABULA_TAGS.PUB_ACTION },
+              title: '.formValues.pubName',
+              tags: '.formValues.tags',
+              description: '.formValues.pubDescription',
+              image: '.formValues.pubImage',
+            },
+          },
+          { type: 'static', value: TABULA_TAGS.PUBLICATION },
+        ],
+      },
+    ],
+  }),
+  CREATE_ARTICLE: buildMultiCallTX({
+    id: 'CREATE_ARTICLE',
+    JSONDetails: {
+      type: 'JSONDetails',
+      jsonSchema: {
+        title: '.formValues.title',
+        description: '.formValues.description',
+        link: '.formValues.link',
+        proposalType: { type: 'static', value: 'Create Article Proposal' },
+      },
+    },
+    actions: [
+      {
+        contract: CONTRACT.POSTER,
+        method: 'post',
+        args: [
+          {
+            type: 'JSONDetails',
+            jsonSchema: {
+              action: { type: 'static', value: TABULA_TAGS.ARTICLE_ACTION },
+              publicationId: '.formValues.pubId',
+              article: { type: 'ipfsPinata', content: '.formValues.article' },
```

Comment:
looks to me like this needs to just be:

```
article: 'hash'
```
https://github.com/gnosis/tabula#create-article